python/tvm/relay/backend/contrib/ethosu/util.py

```diff
@@ -241,6 +241,12 @@ def get_accelerator_config():
     return compiler_attrs.accelerator_config
 
 
+def enable_cascader():
```

Comment:
I think it is better `is_cascader_enabled`, otherwise, it seems you are enabling it with this function.

Reply:
Good point! I changed the function name to `is_cascader_enabled`, but kept the flag/variable as `enable_cascader` to align with the philosophy of `enable_usmp`